butterfly-slack/src/main/java/com/paypal/butterfly/slack/SlackNotifierConfig.java

```
@@ -0,0 +1,121 @@
+            @Override
+            public void postTransformation(TransformationRequest transformationRequest, TransformationResult transformationResult) {
+                postNotification(transformationResult, "postSuccess.json");
+            }
+
+            @Override
+            public void postTransformationAbort(TransformationRequest transformationRequest, TransformationResult transformationResult) {
+                postNotification(transformationResult, "postAbort.json");
+            }
+
+            private void postNotification(TransformationResult transformationResult, String jsonFile){
+                String jsonPath = new File("").getAbsolutePath().concat("/butterfly-slack/src/main/resources/" + jsonFile);
+                String token = System.getProperty("slack.token");
+                try {
+                    String jsonFileString = new String(Files.readAllBytes(Paths.get(jsonPath)));
+                    Client client = ClientBuilder.newClient();
+                    WebTarget target = client.target("https://www.slack.com/api/");
+                    String[] butterflyUserIDs = transformationResult.getUserId().split(" ");
+                    butterflyUserIDs = new String[]{"edwardhwang@berkeley.edu", "ehwang0824@gmail.com"};
+                    String requestID = transformationResult.getId();
+                    for (String id : butterflyUserIDs) {
+                        //TODO: Change hard coded email into butterflyUserIDs
+                        String email = id + ""; //id + "@paypal.com";
+                        String slackUserID = getSlackUserID(target, token, email);
+                        String slackChannelID = getSlackChannelID(target, token, slackUserID);
+                        postSlackMessage(target, token, slackChannelID, jsonFileString, requestID);
+                    }
+                } catch (IOException e) {
+                    logger.warn("No JSON message found for slack notification");
```

Add the exception as the second parameter to this log call. If you don't do it, the exception will be swallowed.